dashboard/modules/job/job_head.py

```diff
@@ -69,6 +71,18 @@ def job_exists(self, job_id: str) -> bool:
         status = self._job_manager.get_job_status(job_id)
         return status is not None
 
+    @routes.get("/api/version")
+    async def get_version(self, req: Request) -> Response:
+        resp = VersionResponse(
+            version=CURRENT_VERSION,
+            ray_version=ray.__version__,
```

Comment:
just for curiosity .. is it possible or supported that user submits a job with a different ray version / commit, such that laptop sdk has different version compare to the remote ray cluster ? It looks like a common case at least at commit level later on.

Reply:
Yes definitely. We should try to maintain compatibility as much as possible. It's a big pain point right now for Ray client that you need to match basically the exact version.